kong/plugins/prometheus/handler.lua

```diff
@@ -34,6 +37,26 @@ function PrometheusHandler.log(self, conf)
     end
   end
 
+  local tags = {}
+  if conf.expose_tags.from_service and message.service and message.service.tags then
+    for i = 1, #message.service.tags do
+      tags[message.service.tags[i]] = true
+    end
+  end
+  if conf.expose_tags.from_route and message.route and message.route.tags then
+    for i = 1, #message.route.tags do
+      tags[message.route.tags[i]] = true
+    end
+  end
+  if conf.expose_tags.from_consumer and message.consumer and message.consumer.tags then
+    for i = 1, #message.consumer.tags do
+      tags[message.consumer.tags[i]] = true
+    end
+  end
+  tags = keys(tags)
+  sort(tags)
+  serialized.tags = concat(tags, ",")
```

Comment:
```suggestion
  table.sort(tags)
  serialized.tags = table.concat(tags, ",")
```

Reply:
this code will collect all the keys from the table (that does not have duplicated strings, as they are the keys), sort it (to avoid any troubles with cardinality) then concatenate it